modules/ROOT/pages/maintenance/backup.adoc

```diff
@@ -0,0 +1,44 @@
+= Backup
+:toc: right
+
+:description: Making a backup of Infinite Scale is an important task that should be done on a regular basis.
+
+== Introduction
+
+{description} This ensures that in case of issues, data can be restored from a backup.
+
+== General Considerations
+
+Infinite Scale can run in two different setups when it comes to storing data.
+
+* All data, means configuration, user data, and metadata is stored on POSIX compliant filesystems.
+* User data is stored as blob on an S3 compliant storage and configuration respectively metadata is stored on POSIX
+
+For details on supported filesystems see: xref:prerequisites/prerequisites.adoc#filesystems-and-shared-storage[Filesystems and Shared Storage]
+
+Depending on how this data is stored, backup strategies and tasks will differ, see the sections below.
+
+For the time being, you have to fully stop the Infinite Scale instance to create a consistant state necessary for taking the backup. In the scenarious shown, this consistancy state is a prerequisite for all upcoming tasks and the way to it is not explicit mentioned anymore. When the backup has finished, the Infinite Scale instance can be started again.
```

Comment:
```suggestion
You have to fully stop the Infinite Scale instance to create a consistent state necessary for performing the backup. In the following scenarios, this state of consistency is a prerequisite for all upcoming tasks. To avoid redundant information, this explanation will not be repeated every time. When the backup is finished, Infinite Scale can be started again.
```

Reply:
The commands to stop and start Infinite Scale should be added here.